src/views/Task/TaskDocuments/TaskDocumentsList.vue

```diff
@@ -45,6 +45,12 @@ function isRunning(item) {
   return item.state === TASK_STATUS.RUNNING
 }
 
+function removeWithConfirmation(id, callback) {
+  afterConfirmation(async () => {
```

Comment:
Because `afterConfirmation` is async, it's important to have a `return` here or the promise will create an "unhandled promise rejection" warning.

```suggestion
  return afterConfirmation(async () => {
```